src/main/scala/firrtl/util/BackendCompilationUtilities.scala

```diff
@@ -91,7 +91,7 @@ trait BackendCompilationUtilities {
     val topModule = dutFile
 
     val blackBoxVerilogList = {
-      val list_file = new File(dir, firrtl.transforms.BlackBoxSourceHelper.FileListName)
+      val list_file = new File(dir, firrtl.transforms.BlackBoxSourceHelper.fileListName)
```

Comment:
Isn't the preferred style to have **constants** in companion objects start with upper case.

Reply:
Upon further evaluation, it is: https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods

That being said, this isn't really a constant since it depends on dir